db/seeds.rb

```diff
@@ -75,7 +75,7 @@
 
 # Vacancies at multiple schools in Southampton local authority
 attrs = { organisations: southampton_la.schools.first(5), phases: %w[primary], publisher_organisation: southampton_la, publisher: Publisher.all.sample }
-3.times { FactoryBot.create(:vacancy, :for_seed_data, :published, **attrs) }
+3.times { FactoryBot.create(:vacancy, :published, **attrs) }
```

Comment:
Is there any reason why this particular org doesn't use the `:for_seed_data` vacancy trait? Isn't the `phases` defined in the `attrs` overriding the one coming from the trait? Or something else?

Reply:
thanks, that looks like a weird git merge issue